internal/cli/cmd/plugin/build/build.go

```diff
@@ -126,7 +287,7 @@ func (o *option) computeArchiveFiles() ([]archives.FileInfo, error) {
 		return nil, fmt.Errorf("unable to read 'schema' directory: %w", err)
 	}
 	for _, f := range cueFiles {
-		list[filepath.Join(o.cfg.SchemasPath, f.Name())] = path.Join("schemas", f.Name())
+		list[filepath.Join(o.cfg.SchemasPath, f.Name())] = filepath.Join("schemas", f.Name())
```

Comment:
I have intentionally let the usage of the pkg `path` here because the value of the map is what will be created in the archive. And I am pretty sure, the path shouldn't be OS relative but the regular `/schemas/cue.mod` one

Reply:
excepting if you found an issue when creating the archive, I believe you should let it like that

Reply:
Ok indeed I thought that was a miss, I'll check if this is not causing any issue

Reply:
seems fine, reverted